src/services/WolfDen.Infrastructure/Data/WolfDenContext.cs

```diff
@@ -18,12 +18,29 @@ public WolfDenContext(DbContextOptions<WolfDenContext> options):base(options) {
         public DbSet<Employee> Employees { get; set; }
         public DbSet<Department> Departments { get; set; }
         public DbSet<Designation> Designations { get; set; }
+       public DbSet<AttendenceClose> AttendenceClose { get; set; }
+        public DbSet<AttendenceLog> AttendenceLog { get; set; }
+       public DbSet<DailyAttendence> DailyAttendence { get; set; }
+       public DbSet<Device> Device { get; set; }
+       public DbSet<Holiday> Holiday { get; set; }
+       public DbSet<Notification> Notification { get; set; }   
+       public DbSet<Status> Status { get; set; }
+        public DbSet<StatusType> StatusType { get; set; }
+
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             modelBuilder.ApplyConfiguration(new EmployeeConfiguration());
             modelBuilder.ApplyConfiguration(new DepartmentConfiguration());
             modelBuilder.ApplyConfiguration(new DesignationConfiguration());
+            modelBuilder.ApplyConfiguration(new AttendenceCloseConfiguration());
```

Comment:
merge with latest , this has been handled generically no need to specify individually

Reply:
updated considering the suggestions and a new pull request is given ,in that it is merged with the latest.